lib/core/presentation/pages/event/create_event/create_event_page.dart

```diff
@@ -35,6 +35,8 @@ class CreateEventPage extends StatelessWidget implements AutoRouteWrapper {
               EventDateTimeSettingsEvent.init(
                 startDateTime: EventDateTimeConstants.defaultStartDateTime,
                 endDateTime: EventDateTimeConstants.defaultEndDateTime,
+                tempStartDateTime: EventDateTimeConstants.defaultStartDateTime,
```

Comment:
as we remove this outside bloc scope, wonder if it's still needed, otherwise let remove